CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing to farmOS
+
+There are many ways to get involved in the farmOS community. You don't need to
+be a programmer. All are welcome to take part in the process.
+
+## Use farmOS
+
+First and foremost: use farmOS.
+
+Building a critical mass of farmOS users is the best way for the project to
+grow and thrive. If you would like to improve your record keeping habits, or
+just want a place to put everything so you can access it easily, farmOS can
```

Comment:
Maybe this should be a tiny bit more specific about the use-case niche for farmOS; "If you would like to improve your **farming/gardening/agricultural** record keeping habits, or just want a place to put everything so you can access it easily, farmOS can help."